code/game/area/areas.dm

```diff
@@ -310,7 +310,7 @@ GLOBAL_LIST_EMPTY(teleportlocs)
 				A.power_light = FALSE
 				A.power_equip = FALSE
 				A.power_environ = FALSE
-			INVOKE_ASYNC(A, .proc/power_change)
+			A.power_change()
```

Comment:
this is invoke async because iterating through the machines in an area is slow, and we do **not** want to clog the gc.

Reply:
this is incorrect, `INVOKE_ASYNC` just sets `waitfor = FALSE` on arbitrary procs, it isn't even remotely similar to a spawn. the name is very misleading. re-adding the `INVOKE_ASYNC` will do nothing but make calling `power_change` here slightly slower.